src/batou/lib/buildout.py

```diff
@@ -58,6 +59,15 @@ def configure(self):
         if self.wheel:
             venv += Package("wheel", version=self.wheel)
 
+        if self.pip:
+            venv = [
+                x for x in self.sub_components if isinstance(x, VirtualEnv)][0]
+            # Remove default `--ignore-installed` which breaks a possibly
+            # requested downgrade of pip as it omits deleting the previously
+            # installed version:
+            venv.venv.install_options = ()
```

Comment:
```suggestion
```
I hope, we can leave that out, as https://github.com/flyingcircusio/batou/commit/41cb58b8d9cb7af4675a51f6b8ed991d37bf6955#diff-e81661b692ea23dd37b493a40c3c5582b2e50ac0858309fe72fc02c9644f4eee was already merged. This seems to be a relict from before that time.